src/node-sim.lisp

```diff
@@ -31,9 +31,21 @@ witnesses."
               (not (and (probe-file cosi-simgen::*default-data-file*)
                         (probe-file cosi-simgen::*default-key-file*))))
     (cosi-simgen::generate-tree :nodes nodes))
-  (setf cosi-simgen::*cosi-prepare-timeout* cosi-prepare-timeout)
```

Comment:
@guitarvydas You've deleted the configuration options to set the Cosi timeouts.  Please reinstate, or explain why you deleted this option.